AUTHORS.rst

```diff
@@ -8,4 +8,4 @@ Contributors
 * Andrew Montanez <amontane@mit.edu>
```

Comment:
Authors should be sorted by contribution ranking, as given by `git shortlog -sn`.
Also, since in this case there are important contributions in the background, not shown as code changes, I would add a *Development Lead* section and move Kalyan Veeramachaneni and Yi Sun there.